src/transformers/integrations/peft.py

```diff
@@ -61,14 +66,15 @@ class PeftAdapterMixin:
 
     def load_adapter(
         self,
-        peft_model_id: str,
+        peft_model_id: Union[str, Dict[str, "torch.Tensor"]],
         adapter_name: Optional[str] = None,
         revision: Optional[str] = None,
         token: Optional[str] = None,
         device_map: Optional[str] = "auto",
         max_memory: Optional[str] = None,
         offload_folder: Optional[str] = None,
         offload_index: Optional[int] = None,
+        peft_config: Dict[str, Any] = None,
```

Comment:
not in docstring

Reply:
[55d657b](https://github.com/huggingface/transformers/pull/25721/commits/55d657b072beddf4080cb2b44c0b33fafb8f7e87)